docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+``values.yaml`` file.
+
+.. code-block:: yaml
+
+    additionalCatalogs:
+      lakehouse.properties: |-
+        connector.name=iceberg
+        hive.metastore.uri=thrift://example.net:9083
+      rdbms.properties: |-
+        connector.name=postgresql
+        connection-url=jdbc:postgresql://example.net:5432/database
+        connection-user=root
+        connection-password=secret
+
+This adds both ``lakehouse`` and ``rdbms`` catalogs to the Kubernetes deployment
+configuration.
+
+.. _running-a-local-kubernetes-cluster-with-kind:
+
+Running a local Kubernetes cluster with kind
+--------------------------------------------
+
+For local deployments, you can optionally use
+`kind (Kubernetes in Docker) <https://kind.sigs.k8s.io>`_. Follow the steps
+below to run ``kind`` on your system.
+
+#. ``kind`` runs on `Docker <https://www.docker.com>`_, so first check if Docker
+   is installed:
+
+   .. code-block:: text
+
+      docker --version
+
+   If this command fails, install Docker by following
+   `Docker installation instructions <https://docs.docker.com/engine/install/>`_.
+
+#. Install ``kind`` by following
+   `kind installation instructions <https://kind.sigs.k8s.io/docs/user/quick-start/#installation>`_.
+
+#. Run a Kubernetes cluster in ``kind`` by running the command:
+
+   .. code-block:: text
+
+      kind create cluster --name trino
+
+   .. note::
+
+      The ``name`` parameter here is optional but is just used to showcase how
+      the namespace is applied in future commands. The cluster name defaults to
+      ``kind`` if no parameter is added. Use ``trino`` to make the application
+      on this cluster obvious.
+
+#. Verify that ``kubectl`` is running against the correct Kubernetes cluster.
+
+   .. code-block:: text
+
+      kubectl cluster-info --context kind-trino
+
+   If you have multiple Kubernetes clusters already configured within
+   ``~/.kube/config``, you need to pass the ``context`` parameter to the
+   ``kubectl`` commands to operate with the local ``kind`` cluster. ``kubectl``
+   uses the
+   `default context <https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration>`_
+   if this parameter isn't supplied. Notice the context is the name of the
+   cluster with the ``kind-`` prefix added. Now you can look at all the
+   Kubernetes objects running on your ``kind`` cluster.
+
+#. Set up Trino by folling the `Running Trino using Helm`_ steps. When running
+   the ``kubectl get all`` command, add the ``context`` parameter.
+
+   .. code-block:: text
+
+      kubectl get all --context kind-trino
+
+#. Run some queries by following the `Executing queries`_ steps.
+
+#. Once you are done with the cluster using kind, you can delete the cluster.
+
+   .. code-block:: text
+
+      kind delete cluster -n trino
+
+Cleaning up
+-----------
+
+To uninstall the helm release, and all the Kubernetes instances deployed with it,
```

Comment:
Helm release 

but what do you mean anyway

would it be better to say

To uninstall Trino from the Kubernetes cluster, run the following command: